### DISCUSSION TITLE
Allow to assign a discussion to specific users

### DISCUSSION THREAD

**Discussion by ceolinwill** [community/community#7485] [Discussions]
### Context

We're trying to use GitHub Discussions as our internal Q&A tool where we ask questions related to our software and team practices. It's a way to encourage asynchronous communication and avoid real-time interruptions over Slack.

### Problem

It's hard to keep track of discussions requiring our attention. We receive a notification when we're mentioned in a discussion but we don't have a way to keep track of discussions that still require attention for a specific person.

For example, I know a PR or issue requires my attention when it's assigned to me. If I go to `Pull Requests -> Assigned`, I can quickly see all pull requests requiring my attention. There is no easy way to do that for Discussions.

### What we do now

We're adopting two workarounds:

- Creating a [notification filter](https://github.com/notifications?query=is%3Adiscussion+reason%3Amention+) using `is:discussion reason:mention`. It helps us to find discussions we were mentioned but we need to open the discussion to find out if an action is required on our part.
- Creating a `Pending Response - Team Member Name` label. For example, if I want to find discussions that require my attention, I can filter them by the `Pending Response - Will` label. It's annoying because we need to go to each repo's discussion page and filter them out.

### What we would like to do

- Assign users to discussions in the same way we assign people to PRs and Issues.
- Have a `Discussions` menu in the navbar in the same way we have a `Pull requests` and an `Issues` menu.
- Have an `Assigned` tab in the `Discussions` page in the same way we have an `Assigned` tab in the [Pull requests](https://github.com/pulls/assigned) and in the [Issues](https://github.com/issues/assigned) page.
- Have a `Mentioned` tab in the `Discussions`page in the same way we have a `Mentions` tab in the [Pull requests](https://github.com/pulls/mentioned) and in the [Issues](https://github.com/issues/mentioned) page.

**Comment by david-mcafee** [comment]
+1 to everything said above 🙂
